app/domain/authentication/authn_jwt/jwt_fetch_provider_uri_signing_key.rb

```diff
@@ -0,0 +1,55 @@
+module Authentication
+  module AuthnJwt
+
+    JwtFetchProviderUriSigningKey = CommandClass.new(
```

Comment:
the `Jwt` prefix is redundant and collides with the command class naming pattern (it should have the action that is being taken)